.github/workflows/python_lint.yml

```
@@ -72,14 +72,8 @@ jobs:
-      # Job-specifc step(s):
+      # Job-specific step(s):
 
-      # For now, we run mypy only on modified files
-      - name: Get changed Python files
-        id: changed-py-files
-        uses: tj-actions/changed-files@v43
-        with:
-          files: "airbyte_cdk/**/*.py"
-      - name: Run mypy on changed files
+      - name: Run mypy
         if: steps.changed-py-files.outputs.any_changed == 'true'
```

actionlint (1.7.4)

property "changed-py-files" is not defined in object type {}

(expression)